src/Core/tests/DeviceTests/Stubs/EditorStub.cs

```diff
@@ -29,7 +29,7 @@ public string Text
 
 		public bool IsTextPredictionEnabled { get; set; } = true;
 
-		public Keyboard Keyboard { get; set; }
+		public Keyboard Keyboard { get; set; } = Keyboard.Default;
```

Comment:
FYI, this was affecting some WinUI keyboard tests due to `VirtualView.Keyboard` was null by default, so I fixed it this way:
https://github.com/dotnet/maui/pull/7371/files#diff-a0ac9a5fd24bb1d27df607dbfd4dec3673c00fd529ab88d6c6edf80c13ec7052R179